urlscan/plugin.spec.yaml

```diff
@@ -141,35 +141,39 @@ connection:
     description: urlscan API key. Not required for the search action
     type: credential_secret_key
     required: false
+    example: a46ec67a-0f2e7-c3879-26ac5d783ea4b8
```

Comment:
Doesn't the key format have 5 fields separated by 4 `-`'s?

Reply:
Use this one: 381cd93b-1946-9c73-1946-c916075eb9a3 (sample)

Reply:
And regen